python/lsst/ts/m2com/mock_model.py

```diff
@@ -1120,3 +1121,134 @@ def handle_position_mirror(self, mirror_position_set_point):
+            result = True
+        elif (power_type == PowerType.Communication) and self.communication_power_on:
+            result = True
+
+        return result
+
+    def get_digital_output(self):
+        """Get the value of digital output.
+
+        Returns
+        -------
+        digital_output : `int`
+            Value of the digital output.
+        """
+
+        digital_output = (
+            DigitalOutput.InterlockEnable.value
+            + DigitalOutput.ResetMotorBreakers.value
+            + DigitalOutput.ResetCommunicationBreakers.value
+        )
+
+        if self.communication_power_on:
+            digital_output += DigitalOutput.CommunicationPower.value
+
+        if self.motor_power_on:
+            digital_output += DigitalOutput.MotorPower.value
+
+        return digital_output
+
+    def get_digital_input(self):
+        """Get the value of digital input.
+
+        Returns
+        -------
+        digital_input : `int`
+            Value of the digital input.
+        """
+
+        digital_input = (
+            DigitalInput.RedundancyOK.value
+            + DigitalInput.LoadDistributionOK.value
+            + DigitalInput.PowerSupplyDC_2_OK.value
+            + DigitalInput.PowerSupplyDC_1_OK.value
+            + DigitalInput.PowerSupplyCurrent_2_OK.value
+            + DigitalInput.PowerSupplyCurrent_1_OK.value
+            + DigitalInput.J1_W9_1_MotorPowerBreaker.value
+            + DigitalInput.J1_W9_2_MotorPowerBreaker.value
+            + DigitalInput.J1_W9_3_MotorPowerBreaker.value
+            + DigitalInput.J2_W10_1_MotorPowerBreaker.value
+            + DigitalInput.J2_W10_2_MotorPowerBreaker.value
+            + DigitalInput.J2_W10_3_MotorPowerBreaker.value
+            + DigitalInput.J3_W11_1_MotorPowerBreaker.value
+            + DigitalInput.J3_W11_2_MotorPowerBreaker.value
+            + DigitalInput.J3_W11_3_MotorPowerBreaker.value
+            + DigitalInput.J1_W12_1_CommunicationPowerBreaker.value
+            + DigitalInput.J1_W12_2_CommunicationPowerBreaker.value
+            + DigitalInput.J2_W13_1_CommunicationPowerBreaker.value
+            + DigitalInput.J2_W13_2_CommunicationPowerBreaker.value
+            + DigitalInput.J3_W14_1_CommunicationPowerBreaker.value
+            + DigitalInput.J3_W14_2_CommunicationPowerBreaker.value
+            + DigitalInput.InterlockPowerReplay.value
+        )
+
+        if self.communication_power_on:
+            digital_input -= (
+                DigitalInput.J1_W12_1_CommunicationPowerBreaker.value
+                + DigitalInput.J1_W12_2_CommunicationPowerBreaker.value
+                + DigitalInput.J2_W13_1_CommunicationPowerBreaker.value
+                + DigitalInput.J2_W13_2_CommunicationPowerBreaker.value
+                + DigitalInput.J3_W14_1_CommunicationPowerBreaker.value
+                + DigitalInput.J3_W14_2_CommunicationPowerBreaker.value
+            )
+
+        if self.motor_power_on:
+            digital_input -= (
```

Comment:
Same as above but you can call these `digital_inputs_motor_power`

Reply:
Fixed in the commit: [8e6d00e](https://github.com/lsst-ts/ts_m2com/pull/2/commits/8e6d00e84f176e5c1227e83f56e9a124aa1afd53). Thanks!